lib/tasks/update-package-json.test.js

```diff
@@ -161,4 +161,20 @@ describe('updatePackageJson() function', () => {
       }
     `);
   });
+
+  it('updates dependencies to the minimum required version (packagesToUpdate)', async () => {
+    files = {
+      'package.json': JSON.stringify({
+        devDependencies: {
+          '@embroider/router': '2.1.8',
+        },
```

Comment:
nitpick: The other tests also seem to test the other key
```suggestion
        devDependencies: {
          '@embroider/router': '2.1.8',
        },
        dependencies: {
          '@embroider/router': '2.1.8',
        },
```

Reply:
The comment makes sense, but the fix is a bit different 🙈 In the other tests, `dependencies` and `devDependencies` have various packages, so the test ensures one package can be found in one place or the other. But if you put the same package in both places, the codemod is not going to like it; this is not supposed to happen.